.github/workflows/base.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Build Base"
+run-name: Build Base of ${{ github.ref_name }} by @${{ github.actor }}
+on:
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        type: boolean
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        required: false
+        default: false
+
+jobs:
+  build:
+    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.12
```

Comment:
can we do 13 here?

Reply:
that's not in Hyku. Trying to keep this inline with Hyku